crane/container.go

```diff
@@ -933,8 +929,10 @@ func (c *container) createArgs(cmds []string, excluded []string) []string {
 		args = append(args, "--env-file", envFile)
 	}
 	// Expose
-	for _, expose := range c.RunParams().Expose() {
-		args = append(args, "--expose", expose)
+	if !adHoc {
```

Comment:
Oh right. I just saw them mentioned in the CLI docs and thought they worked differently, haven't used that option so far. Now after properly reading the doc I see it's fine. Will undo.